app/main.ipynb

```diff
@@ -268,18 +356,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A[:2, :]",
```

Comment:
The selection of the first two rows is correct. Ensure that the slicing indices match the task requirements.